CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(abb_librws)
 
+add_compile_options(-std=c++11)
+
```

Comment:
I don't think that a requirement for C++11 should be added in this PR, and it means that the scoped enums should not be used (see my other comments).

A separate PR should be made for adding C++11 requirement.